mmpose/models/backbones/litehrnet.py

```diff
@@ -1,8 +1,3 @@
-# ------------------------------------------------------------------------------
```

Comment:
Keep these lines. It is for this whole file. The notice in docstring is for documentation.